sensor/kubernetes/certrefresh/certrequester/types.go

```diff
@@ -0,0 +1,21 @@
+package certrequester
```

Comment:
I couldn't come up with a better name for the package and file, open to suggestions. :)

Reply:
The type would be used as `certrequester.CertificateRequester`. The "idiomatic" part of the community would suggest `certificate.Requester`. I don't have better ideas at the moment, but I am only starting with this PR. Will let you know when I get an idea.

Reply:
I thought about that as well, but I thought that a package named "certificate" doesn't make much sense in this context.

Reply:
I renamed `certrequester` to `certificates`, which is close to what you were suggesting. Somehow the plural sounds better to me, because we're always issuing a set of certs. And then I renamed the types of the package to avoid repetition.